test/integration/relay/connection.test.js

```diff
@@ -347,6 +358,96 @@ if (helper.sequelize.dialect.name === 'postgres') {
         expect(lastResult.data.user.tasks.pageInfo.hasNextPage).to.equal(false);
       });
 
+      it('should support in-query slicing and pagination with first and CUSTOM orderBy', async function () {
+        const correctOrder = await graphql(this.schema, `
+          {
+            user(id: ${this.userA.id}) {
+              tasks(first: 9, orderBy: CUSTOM) {
+                edges {
+                  cursor
+                  node {
+                    id
+                    name
+                  }
+                }
+                pageInfo {
+                  hasNextPage
+                  endCursor
+                }
+              }
+            }
+          }
+        `);
+        const reordered = correctOrder.data.user.tasks.edges.map(({node}) => {
+          const targetId = fromGlobalId(node.id).id;
+          return this.userA.tasks.find(task => {
+            return task.id === Number(targetId);
+          });
+        });
```

Comment:
The rest of the test tests that paging works against the first result, however we haven't exactly verified the first result :) 
Isn't stricly necessary but does increase robustness.